react/src/components/calculators/directors/DirectorsForm.tsx

```diff
@@ -123,6 +136,26 @@ export default function DirectorsForm(props: DirectorsFormProps) {
         error={errors.earningsPeriod}
       />
 
+      {askApp && <Radios
+        legend="Is an Appropriate Personal Pension Scheme applicable?"
```

Comment:
this is probably more accurately described by something like `Did the director have an Appropriate Personal Pension or Appropriate Personal Pension Stakeholder Pension in this earnings period?`

Reply:
updated to use this new legend